Sources/Rendering/Core/Camera/index.js

```diff
@@ -348,7 +348,26 @@ function vtkCamera(publicAPI, model) {
   publicAPI.getFrustumPlanes = (aspect) => {
     // Return array of 24 params (4 params for each of 6 plane equations)
   };
-  publicAPI.getCameraLightTransformMatrix = () => {};
+
+  publicAPI.getCameraLightTransformMatrix = () => {
+    const matrix = mat4.create();
+    mat4.copy(matrix, model.cameraLightTransform);
+    return matrix;
+  };
+
+  publicAPI.computeCameraLightTransform = () => {
+    // not sure if this is the correct transformation, based on the same funciton in VTK
+    mat4.copy(tmpMatrix, publicAPI.getViewMatrix());
+    mat4.invert(tmpMatrix, tmpMatrix);
+    mat4.scale(tmpMatrix, tmpMatrix, [
+      model.distance,
+      model.distance,
+      model.distance,
+    ]);
+    model.cameraLightTransform = mat4.create();
+    mat4.translate(model.cameraLightTransform, tmpMatrix, [0.0, 0.0, -1.0]);
```

Comment:
Let's create this in the constructor and then we can identity() it here. That way we are not creating it on the heap every frame when the camera is moving.